python/lsst/meas/algorithms/__init__.py

```diff
@@ -70,3 +70,7 @@
 for name in dict(globals()):
     if name.endswith("_swigregister"):
         del globals()[name]
+
+from lsst.pex.config import makeConfigClass
+makeConfigClass(CoaddPsfControl, "CoaddPsfConfig")
+del makeConfigClass
```

Comment:
While it's a bigger change, I think it'd be more appropriate to move this to `coaddPsf/coaddPsfContinued.py` (and move `coaddPsf.cc` to `coaddPsf/coaddPsf.cc` and add an `__init__.py` there).

And while it's unrelated, would you mind zapping the Swig cleanup code just above this block?